scripts/ssl/sslScanParser.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+import json
+import sys
+import os
+import traceback
+
+
+with open(str(sys.argv[1]), 'r') as data_file:
+    data = json.load(data_file)
+orgName = str(sys.argv[1]).split('.json', 1)[0]
+if data[0]['status'] == 'ERROR':
+    print "sslscan finished with errors"
+    sys.exit(0)
+print "RESULT IN SSL_TLS_TESTING_FUNCTIONALITY_FROM_SSLLABS_REPORT.html"
+
+try:
+
+    f = open(orgName+'_report.html', 'w')
+    f.write('<html>')
+    f.write('<body>')
+    f.write('<div style="background-color: lightgrey;border:1px solid;border-radius:10px;width:300px;"')
+    f.write('<h1>SSL SCAN REPORT</h1>')
+    f.write('</div>')
+    f.write('<h2> Input Url :'+data[0]['host']+'</h2>')
+    f.write('<h2> Domain :'+data[0]['host']+'</h2>')
+    f.write('<h2' + str(data[0]['endpoints'][0]['details']['httpStatusCode'])+'</h2>')
+    f.write('<h2> IP :'+data[0]['endpoints'][0]['ipAddress']+'</h2>')
+    f.write('<h2> Grade :'+data[0]['endpoints'][0]['grade']+'</h2>')
+    f.write('<h2> Secondary Grade :'+data[0]['endpoints'][0]['gradeTrustIgnored']+'</h2>')
+    f.write('<h2> Freak : '+str(data[0]['endpoints'][0]['details']['freak'])+'</h2>')
```

Comment:
This should be `f.write('<h2> Freak :</h2> '+str(data[0]['endpoints'][0]['details']['freak']))`. This makes it more readable - the values are of different font size for `key:` and `value`.